src/pom.xml

```diff
@@ -2012,6 +2012,7 @@
   <joda-time.version>2.8.1</joda-time.version>
   <git.commit.useNative>false</git.commit.useNative>
   <git.commit.runOnlyOnce>true</git.commit.runOnlyOnce>
+  <eclipse.emf.version>2.15.0</eclipse.emf.version>
```

Comment:
Thanks this looks good and agrees with geotools version in https://github.com/geotools/geotools/pull/3072